src/components/DropDetails.js

```diff
@@ -29,11 +29,10 @@ export const DropDetails = ({ title, startOpen, onToggle, children }) => {
 
   return (
     <div className={`container-drop-div ${open ? 'container-drop-div-open' : ''}`}>
-      <div className="container-drop-header">
+      <div className="container-drop-header" onClick={click}>
```

Comment:
note(non-blocking): This behavior was supposed to be fixed in #378, but an issue with overlays postponed it.

I'm sharing this here because I believe there are still some places where this kind of overlay issue is still happening, and in these cases the `container-drop-header` click event will seem not to be working, breaking the manual QA.

I couldn't find precisely _where_ to open an objective issue about it, but will do it soon.

Reply:
I don't understand. This makes the whole dropdown header row clickable. Is it supposed to be fixed in #378?

Reply:
We currently have some "ghost" divs with a high z-index that block the click behavior of some elements on the screen.

This is the issue that was partially solved in 378. However, every now and then I try to click some element on screen and notice it's not working. Just a small scroll up or down is enough to avoid this problem.

During an eventual QA, this issue might be mistaken for a bug on your code on the click behavior of the header row, and that's why I brought it here: to clarify the actual source of the problem.